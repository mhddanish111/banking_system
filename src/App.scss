@import './styles/colors.scss';
@import './styles/mixins.scss';


/* Main CSS Here */
@import "https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap";
* {
	margin: 0;
	padding: 0;
	box-sizing: border-box;
	font-family: $font-family_1;
}
body {
	background-color: $background-color_3;
	max-width: 100%;
	overflow-x: hidden;
}

.main-container {
	display: flex;
	width: 100vw;
	position: relative;
	top: 70px;
	z-index: 1;
}

.main {
	height: calc(100vh - 70px);
	width: 100%;
	overflow-y: scroll;
	overflow-x: hidden;
	padding: 40px 30px 30px 30px;
	&::-webkit-scrollbar-thumb {
		background-image: linear-gradient(to bottom, rgb(0, 0, 85), rgb(0, 0, 50));
	}
	&::-webkit-scrollbar {
		width: 5px;
	}
	&::-webkit-scrollbar-track {
		background-color: $background-color_7;
	}
	@include tablet {
		padding: 40px 30px 30px 30px;
	}
}

