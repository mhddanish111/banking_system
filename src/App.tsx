import React from 'react';
import Switcher from './switcher';
import AppContextWrapper from './context';
import './App.scss';

function App() {
  return (
    <div className="App">
      <AppContextWrapper>
        <Switcher />
      </AppContextWrapper>
    </div>
  );
}

export default App;
