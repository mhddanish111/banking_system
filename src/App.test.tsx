import { shallow } from 'enzyme';
import App from './App';

describe('<App />', () => {
  const wrapped = shallow(<App />);
  it('renders App Component', () => {
    expect(wrapped).toMatchSnapshot();
  });
});
