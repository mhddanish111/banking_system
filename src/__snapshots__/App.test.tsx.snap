// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders App Component 1`] = `
<div
  className="App"
>
  <AppContextWrapper>
    <Switcher />
  </AppContextWrapper>
</div>
`;
